src/components/Badge/styled.tsx

```diff
@@ -1,7 +1,7 @@
 import styled, { css } from 'styled-components';
 import { em, transparentize } from 'polished';
 
-export const VARIANTS = [
+export const variants = [
```

Comment:
Hmm does this look not like a constant anymore?

Reply:
All of our other constants are lowercase so I changed this one. Could also change the others to be uppercase. 🤔

Reply:
Got it, I think it's fine if others also use lowercase, for consistency. Uppercase looks good to me but just checked and It looks like we need to modify some more files so it might not be that worth for now